src/test/mochitest/head.js

```diff
@@ -1084,6 +1084,47 @@ function getCM(dbg) {
   return el.CodeMirror;
 }
 
+
+function getCoordsFromPosition(cm, { line, ch }) {
+  return cm.charCoords({ line: ~~line, ch: ~~ch });
```

Comment:
I had to lookup this :) So this is done to have a Math.floor, do we ever trigger this with floating numbers ?

Reply:
I believe this helped before and it is copy pasta from elsewhere